lib/item_repository.rb

```diff
@@ -0,0 +1,29 @@
+require "database_connection"
+require "item"
+
+class ItemRepository
+  def all 
```

Comment:
This method could be a bit shorter. Perhaps assigning the id and name etc. could be a separate method